src/utils/logic-app/designerUtils.ts

```diff
@@ -0,0 +1,633 @@
+                    "core/main": "core.all.min",
+                    "draft-js": "draft.min",
+                    "draft-js-export-html": "draft-js-export-html.min",
+                    "draft-js-import-html": "draft-js-import-html.min",
+                    "fuse": "fuse.min",
+                    "immutable": "immutable.min",
+                    "localforage": "localforage.min",
+                    "oauth": "../Workflow/OAuth/OAuthService",
+                    "office-ui-fabric-react": "office-ui-fabric-react.min",
+                    "prop-types": "prop-types.min",
+                    "react": "react.production.min",
+                    "react-dnd": "react-dnd-bundle/react-dnd/dist/ReactDnD.min",
+                    "react-dnd-html5-backend": "react-dnd-bundle/react-dnd-html5-backend/dist/ReactDnDHTML5Backend.min",
+                    "react-dom": "react-dom.production.min",
+                    "react-draft-wysiwyg": "react-draft-wysiwyg.min",
+                    "reselect": "reselect",
+                    "re-reselect": "re-reselect",
+                    "resources": getResourcePath(),
+                    "SwaggerParser": "swagger-parser.min",
+                    "tslib": "tslib"
+                },
+                shim: {
+                    "office-ui-fabric-react": {
+                        "deps": ["prop-types", "react", "react-dom", "tslib"],
+                        "exports": "Fabric"
+                    }
+                }
+            });
+
+            r(["immutable", "react", "react-dom"], (Immutable, React, ReactDOM) => {
+                global.Immutable = Immutable;
+                global.React = React;
+                global.ReactDOM = ReactDOM;
+
+                r(["draft-js"], Draft => {
+                    global.Draft = Draft;
+
+                    r(["SwaggerParser", "resources", "core/main", "oauth"], (SwaggerParser, resources, designercore, OAuth) => {
+                        designercore.requireScriptForEditor("https://ema.hosting.portal.azure.net/ema/Content/${version}/Scripts/EMAExtension/Client/logicappdesigner/monaco/min/vs");
+
+                        const monacoLocale = getMonacoLocale($locale);
+
+                        if (monacoLocale !== "en") {
+                            global.require.config({
+                                "vs/nls": {
+                                    availableLanguages: {
+                                        "*": monacoLocale
+                                    }
+                                }
+                            });
+                        }
+
+                        const oauthService = new OAuth.OAuthPopupService("https://portal.azure.com");
+
+                        const getArmAccessToken = async () => {
+                            return "${authorization}".substring("Bearer ".length);
+                        };
+
+                        const getCurrentTenantId = async () => {
```

Comment:
Figure out how to get `tenantId` from `client` (or elsewhere).

Reply:
Not required for read-only designer since it is only used for configuring connections using service principals.